__tests__/app.test.js

```diff
@@ -129,5 +129,27 @@ describe('app testing', () => {
             })
         })
     })
+    describe('patch article', () => {
+        test('Returns the requested article with the votes key increased by the amount stated in the patch request', () => {
+            const desiredUpdates = {inc_votes : 1};
+            return request(app)
+            .patch('/api/articles/1')
+            .send(desiredUpdates)
+            .expect(200)
+            .then(({ body: {updatedArticle} }) => {
+                expect(updatedArticle.votes).toBe(101)
+            })
+        })
+        test('Returns the requested article with the votes key decreased by the amount stated in the patch request', () => {
```

Comment:
Please also add the following tests:

Reply:
1. Happy path: Check to make sure the updated votes have been committed to the database. Use your GET /articles/id endpoint after you've done the patch - and assert that you expect the votes to have been updated.

Reply:
2. Sad path: What if the article id is the wrong type of data

Reply:
3. Sad path: what if the article id doesn't exist

Reply:
4. sad path: What if the inc_votes value is the wrong data type?